Lib/test/test_pdb.py

```diff
@@ -319,6 +320,8 @@ def test_pdb_breakpoint_commands():
     > <doctest test.test_pdb.test_pdb_breakpoint_commands[0]>(5)test_function()
     -> print(3)
     (Pdb) break
+    (Pdb) commands 10
+    *** Breakpoint number 10 out of range
```

Comment:
Try to add tests for the other cases.  https://github.com/python/cpython/blob/cd0c4f358b934322ca024692dde7dfe7cf0b5fb6/Lib/bdb.py#L474